server/controllers/finance/journal/index.js

```diff
@@ -49,3 +112,21 @@ function list(req, res, next) {
   })
   .catch(next);
 }
+
+/**
+ * PUT /journal/:UUID/reverse
```

Comment:
This comment should be adjusted to reflect the POST request.  This won't block the PR.